src/plugins/intel_gpu/src/graph/fully_connected.cpp

```diff
@@ -173,10 +173,10 @@ kernel_impl_params fully_connected_inst::get_fake_aligned_params(kernel_impl_par
         auto updated_param = orig_impl_param;
         auto input_shape = orig_input_layout.get_partial_shape().to_shape();
         auto input_row_idx = input_shape.size() - 2;
-        input_shape[input_row_idx] = align_to(input_shape[input_row_idx], 16);
+        input_shape[input_row_idx] = align_to(input_shape[input_row_idx], 8);
         auto output_shape = orig_output_layout.get_partial_shape().to_shape();
         auto output_row_idx = output_shape.size() - 2;
-        output_shape[output_row_idx] = align_to(output_shape[output_row_idx], 16);
+        output_shape[output_row_idx] = align_to(output_shape[output_row_idx], 8);
```

Comment:
How was this number decided?

Reply:
As I can see, blocking by batch dimension is more effective in case of alignment to 8, especially in case of small batch sizes